package/upgrade/upgrade_manifests.sh

```diff
@@ -229,6 +229,8 @@ upgrade_rancher() {
   cd $UPGRADE_TMP_DIR/rancher
 
   ./helm get values rancher -n cattle-system -o yaml >values.yaml
+  echo "Patching rancher features"
+  yq e -i '.features= "multi-cluster-management=true,multi-cluster-management-agent=false"' values.yaml
```

Comment:
Could we move these two lines below the `cat values.yaml`. I believe the original intention of presenting values here is to ensure what the current situation is before upgrading Rancher, then we do the patches.